cirq/contrib/quirk/__init__.py

```diff
@@ -14,4 +14,11 @@
 
 """Converts cirq circuits into quirk circuits."""
 
-from cirq.contrib.quirk.export_to_quirk import circuit_to_quirk_url
+from cirq.contrib.quirk.cells import (
+    QuirkQubitPermutationGate,)
```

Comment:
I intend to expose all Gate classes to the outside world and to give them reprs, so that printing out circuit reprs and pasting them into test cases continues to work.

I agree that it could be fused with the permutation gate from contrib and moved into `ops`, but I think that belongs in a later PR.

The reason the gate is carrying its quirk id is because I later intend to allow round tripping of the gates.

Reply:
Optional: File an issue or leave a TODO so we don't forget to merge the gates.

Reply:
Acknowledged. I'm quite unsure how to merge the gates at the moment (one is more of a utility to make permutation gates whereas the other is an arbitrary permutation).